### ISSUE TITLE
dotted zero variant

### ISSUE THREAD

**Issue by effinsky** [pcaro90/hermit#28]
I really dig this font. It's peculiar, and I dig it. I'm not a big fan of the way zero looks. I wonder if we could get a dotted zero variant. I could get to tinkering with it, but I'm no good at it, really.

Thanks for the font!

**Comment by haunt98** [comment]
I'm in for dotter zero :D But it looks like the font is complete.

**Comment by effinsky** [comment]
meaning it's not been patched up in a long time?

**Comment by pcaro90** [comment]
Indeed, I consider the font complete (as in "it really works for me as it is"). However, feel free to open FontForge, or any other editor of your choice, and play with it!

To get a dotted zero, you could just combine the uppercase "O" letter (U+004F) and the middle dot character (U+00B7), copy-pasting both into the new zero. Then, just adjust sizing, spacing, and whatnot.

I just made a quick-and-dirty test, take a look:

[Hermit-Regular-dottedzero.otf.zip](https://github.com/pcaro90/hermit/files/12551925/Hermit-Regular-dottedzero.otf.zip)

**Comment by effinsky** [comment]
ok cool thanks :)
